src/overlays/actors/ovl_En_Si/z_en_si.c

```diff
@@ -104,11 +104,11 @@ void EnSi_GiveToken(EnSi* this, PlayState* play) {
         Flags_SetTreasure(play, chestFlag);
     }
     Item_Give(play, ITEM_SKULL_TOKEN);
-    if (Inventory_GetSkullTokenCount(play->sceneId) >= 30) {
-        Message_StartTextbox(play, 0xFC, NULL);
-        Audio_PlayFanfare(NA_BGM_GET_ITEM | 0x900);
+    if (Inventory_GetSkullTokenCount(play->sceneId) >= SI_SPIDER_HOUSE_TOKENS_REQUIRED) {
+        Message_StartTextbox(play, 0xFC, NULL); // You collected all tokens, curse lifted
+        Audio_PlayFanfare(NA_BGM_GET_ITEM | 0x100 | SFX_FLAG);
```

Comment:
```suggestion
        Audio_PlayFanfare(NA_BGM_GET_ITEM | 0x900);
```
0x900 is correct here, `SFX_FLAG` doesn't apply to sequences. Only sfx (seq 0)